packages/repocop/src/remediation/dependency_graph-integrator/send-to-sns.ts

```diff
@@ -123,18 +123,19 @@ async function sendOneRepoToDepGraphIntegrator(
 
 export function repoIsExempted(
 	repo: Repository,
-	exceptedCustomProperties: github_repository_custom_properties[],
+	exemptedCustomProperties: github_repository_custom_properties[],
+	language: DepGraphLanguage,
 ): boolean {
-	const repoIsExcepted = exceptedCustomProperties.find(
+	const exemptedRepo = exemptedCustomProperties.find(
 		(property) => repo.id === property.repository_id,
 	);
```

Comment:
To ensure that function only returns true if exemption is for the relevant language:
```suggestion
	const exemptedRepo = exemptedCustomProperties.find(
		(property) => repo.id === property.repository_id && language === property.value
	);
```